script/Arcueid/Day 4/04_00_ARC04_6_2B.txt

```diff
@@ -81,27 +81,27 @@ Those things that attacked us in the alleyway, were they sent by your enemy?"
 [sha:0c0b82ada85a18a8bc13371a798d6355f6a3c599]{
 -- Page 476, Offset 6875.
 -- 「差し向けた、というより巡回ね。
-"I wouldn't say they were 'sent' by him so much as being 'on patrol'.
+"I wouldn't say they were 'sent' by him so much as they were 'on patrol'.
 }
 [sha:78218aa489ee437933521463133aa80a5e61aabe]{
 -- Page 476, Offset 6876.
 -- 一定のルートで街を監視するのがアイツらの役割だったんでしょう。その巡回ルートでわたしと志貴が話しこんでいたから、結果的にわたしが居るってバレちゃったみたい」
-I'm guessing their role was to follow a certain route in order to monitor the city. Since we were chatting in a location along their patrol route, it seems the end result is that my presence has been discovered."
+I'm guessing their role was to follow a certain route in order to monitor the city. Since we were chatting at a location along their patrol route, I guess it just so ended up I got found out."
```

Comment:
Editing splice? Looks like `just so happened` and `ended up` collided a bit. 
> Unfortunately, since were we chatting in the middle of their patrol route, it seems I've been found out.

or
> Since we were chatting in the middle of their patrol route, I've ended up alerting them to my presence here.

and change the next line a bit
> By 'them', you mean your enemy?